### DISCUSSION TITLE
Transform on batch of 3D images

### DISCUSSION THREAD

**Discussion by AshStuff** [Project-MONAI/MONAI#7465] [General]
Is there a way to transform on the batch of 3D images rather than a single 3D image ?

**Comment by KumoLiu** [comment]
Hi @AshStuff, thanks for your interest here.
No, we can only transform for each data or a dictionary of the data.
In MONAI, we assume that all the data passed into the transform starts with the channel dim.
https://github.com/Project-MONAI/MONAI/wiki/MONAI-Data-Shape-Assumption
https://github.com/Project-MONAI/tutorials/blob/main/modules/3d_image_transforms.ipynb
